### PR TITLE
Add ability to skip ssl verification with an alert post node

### PR DESCRIPTION
**Pull request by itsHabib**
Fixes: https://github.com/influxdata/kapacitor/issues/532

###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by itsHabib** [comment]
Hello! There's a few comments I have regarding this issue. The first is that there is a test failing for me but I'm not sure I don't think it s due to my changes. The `server_test.go` fails when I'm on `master` or on `itshabib-skip-ssl-ver`.  Here is the output:

The other  is that I would like to add tests to actually send a POST with TLS so that I can assert using the skipSSLVerification property will prevent a panic. The only issue is that I'm not really sure where that should be placed. The three places I see where I can potentially add a test like that is `integrations/{streamer,batcher}_test.go` and `server/server_test.go`. Do you have any recommendations?

**Comment by itsHabib** [comment]
Huh but looking at the CI tests there was no errors. Can it be possible its just my system?

File: services/httppost/service.go [hunk 1]
```diff
@@ -323,8 +329,18 @@ func (h *handler) Handle(event alert.Event) {
 		req = req.WithContext(ctx)
 	}
 
+	// Skip SSL verification?
+	httpClient := http.DefaultClient
+	if h.skipSSLVerification {
+		httpClient.Transport = &http.Transport{
```
Comment (nathanielc) [inline services/httppost/service.go]: This needs to create a new client. It cannot modify the default client safely. 

```
httpClient = &http.Client{
    Transport: //...
}
```

**Comment by CWoople** [comment]
I created a stream tick script with an alert that is posting to my express project from a kapacitor docker container. When the stream alert is triggered I'm getting this error. 
x509: cannot validate certificate  because it doesn't contain any IP SANs

I am using the .skipSSLVerification method within the tick script. Any ideas?
[example.txt](https://github.com/influxdata/kapacitor/files/4013302/example.txt)

**Pull request merged**